### PR TITLE
fix: Update metrics tags from db to project

### PR DESCRIPTION
**Pull request by JigarJoshi**
WIth project abstraction in - this PR takes care of refactoring metric tags from `db` --> `project`. For migration purpose `db` is still kept around. It will be removed next month.

### CODE REVIEW

**Comment by JigarJoshi** [comment]
cli-test is expected to remain broken until we release newer CLI.

File: server/request/request.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func NewRequestEndpointMetadata(ctx context.Context, serviceName string, methodI
 }
 
 func GetGrpcEndPointMetadataFromFullMethod(ctx context.Context, fullMethod string, methodType string, req interface{}) Metadata {
-	db, coll := GetDbAndColl(req)
+	db, coll := GetProjectAndColl(req)
```
Comment (pboros) [inline server/request/request.go:76]: The db remained here.

**Comment by pboros** [comment]
You need to add project here as well: https://github.com/tigrisdata/tigris/blob/main/server/metrics/requests.go and all the functions that returns the tag list in that directory and db is there.

**Review by pboros — APPROVED** [review]

File: server/request/request.go (hunk 1 above)
Comment (JigarJoshi) [reply server/request/request.go]: good catch. fixed it.

**Pull request merged**